src/main/java/de/retest/web/RecheckWebOptions.java

```diff
@@ -30,16 +48,33 @@ public static RecheckWebOptionsBuilder builder() {
 	public static class RecheckWebOptionsBuilder extends RecheckOptionsBuilder {
 
 		private AutocheckingCheckNamingStrategy checkNamingStrategy = new CounterCheckNamingStrategy();
+		private AttributesProvider attributesProvider = YamlAttributesProvider.getInstance();
+		private ScreenshotProvider screenshotProvider = new ScreenshotProvider();
 
 		public RecheckWebOptionsBuilder
 				checkNamingStrategy( final AutocheckingCheckNamingStrategy checkNamingStrategy ) {
 			this.checkNamingStrategy = checkNamingStrategy;
 			return this;
 		}
 
+		public RecheckWebOptionsBuilder attributesProvider( final AttributesProvider attributesProvider ) {
+			this.attributesProvider = attributesProvider;
+			return this;
+		}
+
+		public RecheckWebOptionsBuilder omitScreenshots() {
```

Comment:
I think the word `omit` is mostly used in a negative context (see [here](https://www.lexico.com/en/synonym/omit))

> Leave out or exclude (someone or something), either intentionally or forgetfully.

I would propose `noScreenshots` or `disableScreenshots`.